test/integration/patients/patient/patient-flow.js

```diff
@@ -291,7 +291,8 @@ context('patient flow page', function() {
       .get('.picklist')
       .find('.js-picklist-item')
       .contains('Nurse')
-      .click();
+      .click()
+      .wait('@routePatchAction');
```

Comment:
I'm not sure on this one.. it's a coverage check for the owner changing on an action on a flow..    we see it _very_ rarely..  I have a hard time believing this will fix it, but this route does get hit, and it's not waiting for it..